### ISSUE TITLE
View Mood Details

### ISSUE THREAD

**Issue by EPostill** [CMPUT301F19T02/Vybe#36]
[//]: # (These are comments and are used for reference and will not show up in the PR)

User Requirement: 01.03.01

[//]: # (Please Add labels for type of issue)

Description: As a participant, I want to be able to view the details of past mood events, so I don't need to remember specific details myself.

[//]: # (If an issue is a bug or defect, please list steps to be able to reproduce bug and a screenshot of the defect i.e terminal message)

Story Points: ~~5~~ 1*

Related Issues: Depends on #63 being closed
#10 

Task List

- [x] Create View Details Activity
- [x] Modify Base Classes to be Serializable
- [x] Update Main Activity to Handle Seeing the Details
- [x] Finish work on XML 
- [x] Add Button that returns to Main Activity i.e. button that returns you to your Mood History (make sure to not create a new activity, just call finish() - see Moe for clarification).
- [x] ~~JUnit Tests~~ Testing will be moved into a separate issue.

Notes:
- For this issue only, dropdown selector for Vibe should just be a selector containing the names/labels of a Vibe (i.e. Anger, Sad, Surprise, etc.)

[//]: # (Notes:)
[//]: # (- Testing and Documentation tasks should always be assumed, but only need to be explicitly stated if there is something that ~~does not~~ need modification. For example,)
[//]: # (some Issues/Stories may not require one of Unit or Intent tests, and this should be indicated in the task list.)
[//]: # (- Code Reviews are also always assumed to be required, but an issue should only indicate in the description if a review is not required, but someone still shouldn't)
[//]: # (merge their own pull request)

**Comment by raysarinas** [comment]
Resized this story to be 1 point. Bulk of the work required to have completed user requirement 01.03.01 was done as part of Issue #63 when the code base had been written. 

All that is left is to decide if a return button is necessary, as well as finishing up the details of the XML file associated with viewing the details of a Mood/Vibe Event
